server/routers/reservations.test.js

```diff
@@ -1,6 +1,10 @@
+// jest.unmock('pg');
+
 const express = require('express');
 const request = require('supertest');
 
+jest.mock('../db');
```

Comment:
this is routing the whole folder with ../db?

Reply:
this is basically replacing the real '../db' module with my fake '../db' module (which can be found in the '../db/__mocks/' folder.  I do this so that my reservations router doesn't actually insert anything into the real DB, but just simulates it for testing.  Also, this way I can simulate a failed database entry as well to test the router behavior for that.